cli/lib/exec/spawn.js

```diff
@@ -119,6 +119,9 @@ module.exports = {
           electronArgs.unshift('--no-sandbox')
         }
 
+        // docker containers have 64M by default, sometimes even less
+        electronArgs.push('--disable-dev-shm-usage')
```

Comment:
For some reason recent builds are pressing memory much harder, if I add `--disable-dev-shm` it will crash later.

Reply:
That cures just one symptom and not very well, reverting.